showcase/src/app/welcome/pages/home/_home-page.component.scss

```diff
@@ -1,94 +1,168 @@
-//set the header image
-.about-background {
+.centered-title {
+  text-align: center;
+  font-size: 17px;
+  line-height: 30px;
+  font-weight: 500;
+}
+
+.item-alignment {
   display: flex;
+  flex-direction: column;
+  justify-content: center;
   align-items: center;
-  
-  margin-bottom: 20px;
-  height: 100px;
+  box-sizing: border-box;
+  margin: 16px auto;
+}
+
+.doc-subtitle {
+  margin-left: 25px;
+}
 
-  background-color: mat-color($primary-palette, 900);
-  background-image: url(/assets/stark-core/logo/light/stark_logo_light.svg);
+.stark-logo{
+  background-image: url(/assets/stark-core/logo/dark/stark_logo_dark.svg);
+  width: 250px;
+  height: 100px;
+  background-size: contain;
   background-repeat: no-repeat;
-  background-size: 250px 100px;
-  background-position: center;
+  display: flex;
+  margin: auto;
 }
 
-.light-font {
-  font-weight: normal;
+.core-image {
+  mask-image: url("/assets/images/logo/atom.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($accent-palette, 500);
```

Comment:
If you use palettes in these styles then this file should be renamed adding the `theme` prefix as we do with the others and import it in the `showcase/src/styles/_theme.scss`